src/components/geojson-map.vue

```diff
@@ -11,7 +11,13 @@ except according to the terms contained in the LICENSE file.
 -->
 <template>
   <div v-show="featureCount !== 0" ref="el" class="geojson-map">
-    <div ref="mapContainer" class="map-container" :class="{ opaque: shown }" tabindex="0" :inert="!shown"></div>
+    <div ref="mapContainer" class="map-container" :class="{ opaque: shown }" tabindex="0" :inert="!shown">
+      <div class="control-bar">
```

Comment:
I think it'd be better to make `.control-bar` a sibling of `.map-container` rather than a child. `.control-bar` is positioned relative to the root element, not relative to `.map-container`. I think it should be on the same level as `.count`.